patina_dxe_core/Cargo.toml

```diff
@@ -52,6 +52,7 @@ patina_internal_collections = { path = "../core/patina_internal_collections" }
 env_logger = { workspace = true }
 mockall = { workspace = true }
 serial_test = { workspace = true }
+zerocopy = { workspace = true, features = ["derive"] }
```

Comment:
Should we use `zerocopy_derive` for `dev-dependencies` as well? The inconsistency makes an already confusing trap for how that dependency is managed a little more confusing.

I don't have a strong opinion though. This is fine for simplicity if others prefer that.